app/models/resource.rb

```diff
@@ -36,7 +36,7 @@ def blank_location
   def sync_binary
     unless File.file?(raw_location)
       increment_attempts
-      NB.stream_binaries ? stream_binary : download_binary
+      NB.stream_binaries == 'true' ? stream_binary : download_binary
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.